packages/drawer/README.md

```diff
@@ -338,11 +338,12 @@ const MultipleDrawersComponent = () => {
 
 ### LayoutData
 
-| Prop                 | Type   | Default | Description                                                                                         |
-| -------------------- | ------ | ------- | --------------------------------------------------------------------------------------------------- |
-| id                   | string |         | The required id of the layout. This is used to open the `Drawer` with `openDrawer(id)`.             |
-| title _(optional)_   | string |         | The title of the `Drawer`. This is not required if the `Toolbar` item should not open a `Drawer`.   |
-| content _(optional)_ | string |         | The content of the `Drawer`. This is not required if the `Toolbar` item should not open a `Drawer`. |
+| Prop                    | Type    | Default | Description                                                                                                                                                                                         |
+| ----------------------- | ------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| id                      | string  |         | The required id of the layout. This is used to open the `Drawer` with `openDrawer(id)`.                                                                                                             |
+| title _(optional)_      | string  |         | The title of the `Drawer`. This is not required if the `Toolbar` item should not open a `Drawer`.                                                                                                   |
+| content _(optional)_    | string  |         | The content of the `Drawer`. This is not required if the `Toolbar` item should not open a `Drawer`.                                                                                                 |
+| scrollable _(optional)_ | boolean | `true`  | Determines whether the drawer content should have its own scroll container with padding. When false, the content area will not have padding or scroll behavior, allowing full-width/height content. |
```

Comment:
nit: might be worth documenting how scrolling the main page content interacts with the drawer (i believe it's fixed, so will never scroll with the page, right?)